### DISCUSSION TITLE
Question about log(2) in 2D LGCP distance sampling example

### DISCUSSION THREAD

**Discussion by IJCorreia** [inlabru-org/inlabru#238] [Q&A] [answered]
Dear, Finn,

I was checking [this](https://inlabru-org.github.io/inlabru/articles/2d_lgcp_distancesampling.html) example and here:
![image](https://github.com/inlabru-org/inlabru/assets/50990866/17fc7478-2465-4371-9a2e-3881d4d9e5e8)
I assume that to take into account "the unknown direction of the detections" means left vs right of the centre line, but I don't understand what it means to add specifically "Log(2)"...

**Comment by finnlindgren** [comment, marked as answer]
The point process intensity is lambda = exp(eta), where eta is the given predictor expression. Adding log(2) to eta doubles the intensity lambda, which, as you said, is needed to take into account that only the absolute value of the distance is available, and not the direction; the two sides have been superimposed on each other, in effect doubling the point intensity.
